src/spline.rs

```diff
@@ -51,7 +54,7 @@ impl<T, V> Spline<T, V> {
   where
     T: PartialOrd,
   {
-    let mut spline = Spline(keys);
+    let mut spline = Spline(VecDeque::from(keys));
```

Comment:
To not change the public API, I kept things exactly as they were from the outside.
The vector passed to `from_vec` is converted into a deque internally.

Reply:
Seems fair.

Reply:
Should we change the API, now that I think about it, and use something like `impl Into<VecDeque<Key<T, V>>>`?

Reply:
This should be 100% compatible with the old versions anyway, so it's totally fine imo.